tests/trading/test_fillOrder.py

```diff
@@ -154,28 +118,12 @@ def test_fill_order_with_shares_escrowed_sell_with_shares(contractsFixture, cash
     assert noShareToken.balanceOf(tester.a2) == fix(1)
 
     # create order with shares
-    logs = []
-    captureFilteredLogs(contractsFixture.chain.head_state, contractsFixture.contracts['Augur'], logs)
     orderID = createOrder.publicCreateOrder(ASK, fix(1), 6000, market.address, YES, longTo32Bytes(0), longTo32Bytes(0), "42", sender=tester.k1)
     assert orderID
 
     # fill order with shares
     assert fillOrder.publicFillOrder(orderID, fix(1), "43", sender=tester.k2) == 0
 
-    assert len(logs) == 8
-    log1 = logs[7]
-
-    assert log1["_event_type"] == "OrderFilled"
-    assert log1["filler"] == bytesToHexString(tester.a2)
-    assert log1["numCreatorShares"] == fix(1)
-    assert log1["numCreatorTokens"] == 0
-    assert log1["numFillerShares"] == fix(1)
-    assert log1["numFillerTokens"] == 0
-    assert log1["marketCreatorFees"] == fix(1, 10000) / market.getMarketCreatorSettlementFeeDivisor()
-    assert log1["reporterFees"] == fix(1, 10000) / universe.getOrCacheReportingFeeDivisor()
-    assert log1["shareToken"] == market.getShareToken(YES)
-    assert log1["tradeGroupId"] == stringToBytes("43")
-
```

Comment:
I don't see any new log code to replace this and some of the removed code below. Should all of this be getting removed?

Reply:
I'm not sure why it was ever being done this way to begin with since we confirm the on chain results anyway. We only need to confirm logging works once IMO. It's also really expensive to test time wise.